framework/test_app/runners/k8s/k8s_base_runner.py

```diff
@@ -299,6 +299,34 @@ def _reuse_namespace(self) -> k8s.V1Namespace:
         logger.info("Reusing namespace: %s", self.k8s_namespace.name)
         return self.k8s_namespace.get()
 
+    def _create_pod_monitoring(self, template, **kwargs) -> None:
+        if not kwargs["namespace_name"]:
+            raise _RunnerError(
+                "namespace_name required to create PodMonitoring resource"
+            )
+        if not kwargs["deployment_id"]:
+            raise _RunnerError(
+                "deployment_id required to create PodMonitoring resource"
+            )
+        if not kwargs["pod_monitoring_name"]:
+            raise _RunnerError(
+                "pod_monitoring_name required to create PodMonitoring resource"
+            )
+        pod_monitoring = self._create_from_template(
+            template, custom_object=True, **kwargs
+        )
```

Comment:
```suggestion
    def _create_pod_monitoring(
        self,
        template,
        *,
        namespace_name: str,
        deployment_id: str,
        pod_monitoring_name: str,
        **kwargs,
    ) -> None:
        pod_monitoring = self._create_from_template(
            template,
            custom_object=True,
            namespace_name=namespace_name,
            deployment_id=deployment_id,
            pod_monitoring_name=pod_monitoring_name,
            **kwargs,
        )
```

Reply:
Fixed. Thanks for the suggestion starter.